Hotfix/testing omnibus

Snugug opened the pull request:
Omnibus of fixes

* Removes landing page from group of topics in a section
* Removes hover on main nav
* Reverts subnav hide based on a11y team feedback
* Moves Storage-stored images to Cloud Storage Bucket URLs and updates to allow for caching
* Fixes deploy previews and Lighthouse results

Snugug committed:
🐛 (ux) Remove landing page from list of section topics

The landing page itself doesn't belong in the list of sections on the landing page. Stays in the
subnav, though.

Snugug committed:
🐛 (ux) Remove hover for main nav

Wound up being confusing due to the combinatino of items with and without subnav, and determined
that only opening and closing on click was a better UX

Snugug committed:
♻️ (code) Use Google Buck URLs for external images

CORS is likely to only work with these URLs, not the Firebase ones

Snugug committed:
♻️ (code) Use Google Buck URLs for external images

CORS only works with these URLs, not the Firebase ones. Needed so we can
cache external images

Snugug committed:
Merge branch 'hotfix/testing-omnibus' of github.com:chromeos/chromeos.dev into hotfix/testing-omnibus

Snugug committed:
👽 (tests) Update to channel deploys

Previous Firebase Functions previews are deprecated in favor of channels

Snugug committed:
💚 (tests) Use actual exported variable

Snugug committed:
Merge branch 'trunk' into hotfix/testing-omnibus

Snugug committed:
♻️ (ux) Fully hide subnav, not just visually

Originally our a11y team thought visually hiding would be better to allow AT users to navigate
subnav by links, but after testing the visually hidden solution, they thought that it not matching
the screen's visuals was a worse UX. Returning to fully hiding subnav to remove this disconnect.

Snugug committed:
💚 (code) Don't fail CI if Lighthouse fails

If the Lighthouse task fails, CI should still pass

MattMonasch reviewed (dismissed):
LGTM! Style question wrt to the new filter, but it functionally all looks good! :D
Thanks for getting this all together!

Snugug committed:
🔥 (code) Remove Chapters filter

Wound up only needing to slice the first item, so can do this directly in the template. From code
review.

Snugug committed:
♻️ (tests) Only want the Lighthouse task to continue-on-error

MattMonasch reviewed (approved):
LGTM!
